analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RuleChecks/EmptyCollectionsShouldNotBeEnumeratedBase.cs

```diff
@@ -151,4 +155,11 @@ public override void ExecutionCompleted()
 
     private static bool IsEmptyArray(IOperation operation) =>
         operation.AsArrayCreation()?.DimensionSizes.Any(x => x.ConstantValue.Value is 0) ?? false;
+
+    private static ISymbol DictionaryWithAccessedSetter(IOperation operation) =>
+        operation.AsAssignment() is { } assignment
+            && assignment.Target.AsPropertyReference() is { Property.IsIndexer: true } propertyReference
+            && propertyReference.Instance.Type.DerivesOrImplements(KnownType.System_Collections_Generic_IDictionary_TKey_TValue)
```

Comment:
Whitespace is off, should be -4. Only the `?` and `:` lines should have the indent as they have
```suggestion
        && assignment.Target.AsPropertyReference() is { Property.IsIndexer: true } propertyReference
        && propertyReference.Instance.Type.DerivesOrImplements(KnownType.System_Collections_Generic_IDictionary_TKey_TValue)
```